acceptance/setup/pre_suite/50_install_modules.rb

```diff
@@ -4,4 +4,8 @@
 
 step "Install the puppetdb module and dependencies" do
   on master, "puppet module install puppetlabs/puppetdb"
+  if options[:type] == 'aio'
+    # patch puppetdb module for version 4.1.0
```

Comment:
@johnduarte we'll assign it later on, you don't need to pre-assign thats not how we work. But more so, even the module bug belongs in our queue, not in the modules queue. Listen, I'll clean this up for you.